src/client/settings/settingsReducer.js

```diff
@@ -53,6 +56,7 @@ const settings = (state = initialState, action) => {
         showNSFWPosts: action.payload.showNSFWPosts,
         upvoteSetting: action.payload.upvoteSetting,
         rewardSetting: action.payload.rewardSetting,
+        useBeta: !!action.payload.useBeta,
```

Comment:
isn't `useBeta` always a boolean? so you won't need the `!!`

Reply:
It comes from SC2 so it might be `undefined` as well. This way it won't safe as `undefined` in state.

Reply:
ahh I see, makes sense